src/test/java/resources/getTxn.feature

```diff
@@ -13,7 +13,7 @@ Feature: Get Txn Req API test
     Scenario: GTX-002 Get Txn Req API with Params
       Given I have tenant as "gorilla"
       And I call collection api with expected status 200
-      When I call the auth endpoint with username: "mifos" and password: "password"
+      When I call the operations-app auth endpoint with username: "mifos" and password: "password"
```

Comment:
Redundant part of TC as auth-n is disabled for the service.

Reply:
This PR doesn't focuses on editing existing test cases, I only renamed a part of the step in the entire code base. But still if it is fine in context of this PR then I will do it. 
So just let me know if you want me to remove the auth step in all the test cases.

Reply:
And by the way, all the auth step is executed only when authentication is enabled in the code via the application.yaml configuration.